src/main/java/com/patientpal/backend/post/controller/BoardController.java

```diff
@@ -0,0 +1,70 @@
+package com.patientpal.backend.post.controller;
+
+import com.patientpal.backend.member.domain.Member;
+import com.patientpal.backend.member.domain.Role;
+import com.patientpal.backend.member.service.MemberService;
+import com.patientpal.backend.post.domain.Post;
+import com.patientpal.backend.post.dto.*;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Using the '.*' form of import should be avoided - com.patientpal.backend.post.dto.*.